### PR TITLE
feat(molecule/inputTags): add a prop to avoid entering the same tag more than once

### PR DESCRIPTION
**Pull request by a-ferrante**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Only
- [ ] Refactor

### Description, Motivation and Context
We have the requirement to use this component in a way that we could ensure that the user is not able to enter the same value more than once.
Added a prop with default value true to avoid messing up with current implementation and add a check that if the new value is not duplicate its pushed into the tags array, if not the event its sent with the previous values
Not pretty sure about the naming on the prop/func so any input is welcome

### CODE REVIEW

**Review by adriadoz — APPROVED** [review]
IMHO the name is okey!

File: components/molecule/inputTags/src/index.js [hunk 1]
```diff
@@ -63,7 +69,10 @@ const MoleculeInputTags = ({
     const {name} = ev.target
     ev.preventDefault()
     if (value) {
-      const tags = [...tagsFromProps, value]
+      const tags = [...tagsFromProps]
+      if (allowDuplicates || (!allowDuplicates && !isDuplicate(tags, value))) {
```
Comment (tonyghiani) [inline components/molecule/inputTags/src/index.js]: I think you can change this condition to 
```js
if (allowDuplicates || !isDuplicate(tags, value))) {
```
it should work anyway

Comment (a-ferrante) [reply components/molecule/inputTags/src/index.js]: @tonyghiani yes, done 😄

**Review by manjarcode — APPROVED** [review]

**Comment by andresin87** [comment]
Add the allowDuplicates to the readme plz

**Review by ferransimon — COMMENTED** [review]
@a-ferrante What about passing a filter func prop that gets the value and tags that by default always returns true but you can override it so it filters out what you need, some cases can be duplicates but some not, I think it gives more flexibility than just a boolean prop. What do you think?

**Comment by a-ferrante** [comment]
@ferransimon i think is a good idea, but i also think that not allowing entering duplicates its a common case and if by default the function provided always returns true all verticals that want to filter by duplicate will have to define the filter function to do the same thing. I think that if in the future another filter requirment come up we can iterate over this

**Review by ferransimon — APPROVED** [review]

**Review by andresin87 — APPROVED** [review]

**Pull request merged**